### ISSUE TITLE
@angular-builders/custom-webpack to support Angular 16

### ISSUE THREAD

**Issue by CarlosTorrecillas** [just-jeb/angular-builders#1355]
**Is your feature request related to a problem? Please describe.**
`@angular-builders/custom-webpack` compatible with Angular 16

**Describe the solution you'd like**
`@angular-builders/custom-webpack` to be able to be installed on an Angular 16 project

**Describe alternatives you've considered**
I haven't found any yet

**Additional context**
Running the migration of one of my projects from Angular 15 to Angular 16 and leaving `@angular-builders/custom-webpack` on the latest version, produces the following error: 

`This version of CLI is only compatible with Angular versions ^15.0.0, but Angular version 16.0.0 was found instead.` which stops the process during my build

**Comment by CarlosTorrecillas** [comment]
Me too @phihochzwei , hopefully a new version of the package gets released soon. Fingers crossed!!

**Comment by just-jeb** [comment]
Hopefully next week guys, a bit of patience 🙂.

**Comment by phihochzwei** [comment]
> Hopefully next week guys, a bit of patience 🙂.

Well, I guess, ever Angular developer is currently like

![7kkza6](https://user-images.githubusercontent.com/10193885/236335901-409d013a-186c-4455-adfc-7bbbca79dfd4.jpg)

**Comment by simeyla** [comment]
Can anyone comment on the long term viability / prospects for `@angular-builders/custom-webpack` ?
Is it likely to be obsolete / incompatible with any upcoming Angular integrations of vite or esbuild?

Personally I use it for:
- retry plugin (webpack-retry-chunk-load-plugin)
- reduce moment js size (moment-locales-webpack-plugin)
- custom plugin to write out the webpack manifest (to preload JS bundles)

I'm asking more from a technical standpoint rather than a willingness-to-maintain standpoint.

**Comment by polhek** [comment]
@simeyla in our application we use it as we are migrating from angularJS to angular.

**Comment by just-jeb** [comment]
@simeyla as long as Angular supports Webpack this builder will be maintained and probably relevant.  
Native Angular `esbuild` builder doesn't support extension interface at the moment so there is nothing we (or as a matter of fact anyone else) can do about it. If they'll open it for extensions we can create a new builder (like `custom-esbuild`) but it will certainly not be implemented as part of `custom-webpack` builder.  
Hope it answers your question. Related issue: https://github.com/just-jeb/angular-builders/issues/1288

**Comment by Emmanuel-FSE** [comment]
I am waiting for angular-builders/custom-webpack to support angular 16 eagerly, I have an angular project that is stuck because I need do enable some experiments.
Any update on the progress would be much appreciated.

**Comment by just-jeb** [comment]
@Emmanuel-FSE This is the PR to track: https://github.com/just-jeb/angular-builders/pull/1357
I'm waiting on final approval from @arturovt, who's been working on this PR, in order to merge it.

**Comment by md13381** [comment]
> @Emmanuel-FSE This is the PR to track: #1357 I'm waiting on final approval from @arturovt, who's been working on this PR, in order to merge it.

Is he not waiting on you to review it for merge? Its been requested for review for the last couple of days

**Comment by just-jeb** [comment]
@md13381 yeah haha. We've been talking internally, don't worry 😁 It's almost done.

**Comment by just-jeb** [comment]
You're welcome to try the new version under the `@next` tag. We'll give it a few days in beta and then graduate to stable. 
![image](https://github.com/just-jeb/angular-builders/assets/9823087/ceb3e15d-c3a4-45d2-9c3c-d292df5b5d33)
